### PR TITLE
VACMS-5204 Add logging for Broken links

### PR DESCRIPTION
**Pull request by indytechcook**
## Description

See #5204. 

## Testing done

Ran content release and looked at output.

image

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: docroot/modules/custom/va_gov_build_trigger/src/Plugin/AdvancedQueue/JobType/WebBuildJobType.php [hunk 1]
```diff
@@ -118,13 +132,41 @@ public function process(Job $job) {
...
+
+    //$contents = '{"summary":"*`campaign-mission-act`* : \n```<a class=\"vads-c-action-link--blue\" href rel=\"noreferrer noopener\" target=\"_blank\" id=\"a85fcc36ed81ab70421ca9977c5ec256\">\n                See more stories\n              </a>```\n```<a href=\"/outreach-and-events/events/veterans-town-hall-in-pittsburgh-pa-0\" id=\"8987ac5f258d32380703c3678a8b3a15\">\n                      Veterans town hall in Pittsburgh, PA\n                    </a>```\n```<a href=\"/outreach-and-events/events/facebook-live-community-care-info-session-0\" id=\"0dc1302ad74844b2fbdc726eeb672b6c\">\n                      Facebook Live: Community care info session\n                    </a>```","isHomepageBroken":false,"brokenLinksCount":3}';
+    $contents = file_get_contents($path);
+    $json = Json::decode($contents);
+
+    if ($json['brokenLinksCount'] && $json['summary']) {
+      $this->logger->info("There are {$json['brokenLinksCount']} broken links");
+      $this->logger->info(BlazyMarkdown::parse($json['summary']));
+      $this->logger->info(nl2br(BlazyMarkdown::parse($json['summary'])));
+      $this->logger->info(BlazyMarkdown::parse($json['summary'], FALSE));
```
Comment (acrollet) [inline docroot/modules/custom/va_gov_build_trigger/src/Plugin/AdvancedQueue/JobType/WebBuildJobType.php:167]: Are the repeated lines intended?

Comment (indytechcook) [reply docroot/modules/custom/va_gov_build_trigger/src/Plugin/AdvancedQueue/JobType/WebBuildJobType.php:167]: Just testing different things

**Comment by indytechcook** [comment]
A few different rendering styles

image

image

image

image

**Comment by indytechcook** [comment]
Here is using the php-markdown library instead of common mark:

image

**Review by acrollet — APPROVED** [review]

**Pull request merged**